integration/spark/shared/src/main/java/io/openlineage/spark/agent/facets/builder/DatabricksEnvironmentFacetBuilder.java

```diff
@@ -5,8 +5,8 @@
 
 package io.openlineage.spark.agent.facets.builder;
 
-import com.databricks.sdk.scala.dbutils.DbfsUtils;
-import com.databricks.sdk.scala.dbutils.MountInfo;
+import com.databricks.sdk.scala.dbutils.DbfsUtils; // Updated lines
+import com.databricks.sdk.scala.dbutils.MountInfo; // Updated lines
```

Comment:
@algorithmy1 I'm curious how these two comments helped solve the issue - could you give more details?